aiida/restapi/translator/node.py

```diff
@@ -596,6 +597,7 @@ def get_node_shape(ntype):
                     "displaytype": display_type,
                     "group": "inputs",
                     "description": description,
+                    "linklabel": linklabel,
```

Comment:
I don't think the linklabel should be attached to the nodes, but to the edges

Reply:
This information is used in materials cloud when user mouse hover on node. So it needs to be passed in node.